source/hotkeys_.d

```diff
@@ -133,6 +141,14 @@ string defaultName(Bindable bindable){
 		case moveBackward: return "Move Backward";
 		case turnLeft: return "Turn Left";
 		case turnRight: return "Turn Right";
+		case moveUp: return "Move Up";
+		case moveDown: return "Move Down";
+		case cameraForward: return "Camera Move Forward";
+		case cameraBackward: return "Camera Move Backward";
+		case cameraUp: return "Camera Move Up";
+		case cameraDown: return "Camera Move Down";
+		case cameraLeft: return "Camera Move Left";
+		case cameraRight: return "Camera Move Right";
```

Comment:
should be moved to `SacEngine extensions` section

Reply:
done.